<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(68, 68, 68);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="start_but">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>191</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border-radius: 10px;
border-style: outset;
padding: 5px;
background-color: rgb(56, 56, 56);
color: rgb(255, 255, 255)
}

QPushButton:hover {
background-color: rgb(30, 30, 30)
}

QPushButton:pressed {
    border-width: 4px;
    border-style: solid;
	border-color: rgb(0, 210, 255);
border-style: inset;
background-color: rgb(30, 30, 30)
}

QPushButton:checked {
    border-width: 4px;
    border-style: solid;
	border-color: rgb(0, 210, 255);
border-style: inset;
background-color: rgb(30, 30, 30)
}</string>
    </property>
    <property name="text">
     <string>start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stop_but">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>191</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border-radius: 10px;
border-style: outset;
padding: 5px;
background-color: rgb(56, 56, 56);
color: rgb(255, 255, 255)
}

QPushButton:hover {
background-color: rgb(30, 30, 30)
}

QPushButton:pressed {
    border-width: 4px;
    border-style: solid;
	border-color: rgb(0, 210, 255);
border-style: inset;
background-color: rgb(30, 30, 30)
}

QPushButton:checked {
    border-width: 4px;
    border-style: solid;
	border-color: rgb(0, 210, 255);
border-style: inset;
background-color: rgb(30, 30, 30)
}</string>
    </property>
    <property name="text">
     <string>stop</string>
    </property>
   </widget>
   <widget class="QLabel" name="count">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>40</y>
      <width>401</width>
      <height>161</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <pointsize>48</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(86, 86, 86);</string>
    </property>
    <property name="text">
     <string>Null</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="status">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>260</y>
      <width>401</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <pointsize>48</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(186, 186, 186);</string>
    </property>
    <property name="text">
     <string>Null</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>450</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Medium</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(200, 200, 200);</string>
    </property>
    <property name="text">
     <string>Progress</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="batteryBar">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>450</y>
      <width>421</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Medium</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar {
    border: 2px solid grey;
    border-radius: 5px;
	color: rgb(255, 255, 255);
}

QProgressBar::chunk {
	background-color:rgb(93, 230, 114);
    width: 20px;
}</string>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
    <property name="textDirection">
     <enum>QProgressBar::TopToBottom</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>780</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
